www/layouts/about/section.html

```diff
@@ -599,24 +602,23 @@ <h4>Jeffrey S. Ravel</h4>
           Professor of History
         </div>
         <div class="pb-4">
-          <h4>Anjali Sastry</h4>
-          Senior Lecturer, Sloan School of Management
+          <h4 class="color-highlight-red">Ex-Officio Members</h4>
         </div>
         <div class="pb-4">
-          <h4>Michael Short</h4>
-          Associate Professor, Department of Nuclear Science and Engineering
+          <h4>Curt Newton</h4>
+          Director, MIT OpenCourseWare
         </div>
         <div class="pb-4">
-          <h4 class="color-highlight-red">Ex-Officio Members</h4>
+          <h4>Eric Grimson</h4>
+          Vice President for Open Learning, MIT Chancellor for Academic Advancement
         </div>
         <div class="pb-4">
-          <h4>Vijay Kumar</h4>
-          Associate Dean for Open Learning; Executive Director, Abdul Latif
-          Jameel World Education Lab (J-WEL)
+          <h4>Anjali Sastry</h4>
+          Faculty Director of J-WEL, Associate Dean for Open Learning
         </div>
         <div class="pb-4">
-          <h4>Curt Newton</h4>
-          Director, MIT OpenCourseWare
+          <h4>Christopher Capozzola</h4>
+          Senior Associate Dean for Open Learning, Professor of History
```

Comment:
I don't really think these should all be header tags. (Check out the outline of this page... https://gsnedders.html5.org/outliner/process.py?url=https%3A%2F%2Focw.mit.edu%2Fabout%2F)

But this PR is probably not the best place to address that. I can make an issue.